.circleci/workflows.yml

```diff
@@ -1189,10 +1189,6 @@ commands:
           command: |
             source ./scripts/ensure-node.sh
             yarn build --scope cypress
-      - run:
```

Comment:
This runs as a postbuild script, so it will run in the previous step. No need to run it again